osgar/drivers/rosmsg.py

```diff
@@ -427,6 +427,13 @@ def parse_topic(topic_type, data):
         # __slots__ = ['score','calls']
         # _slot_types = ['int32','int32']
         return list(struct.unpack_from('<II', data, pos))  # score and calls
+    elif topic_type == 'std_msgs/String':
```

Comment:
this is already in master ... i.e. strange github diff

Reply:
How do you mean? Where do you see it before this PR? Or do you see it twice when looking at this PR?